### DISCUSSION TITLE
VideoDoorBell einrichten

### DISCUSSION THREAD

**Discussion by sunchaser360** [thkl/hap-homematic#484] [Q&A]
Hallo,

ich bin gerade dabei die Videodoorbell einzurichten.
Ich habe es auch geschafft FFmpeg auf meine CCU3 zu bringen. Ich habe mir die notwendigen URLs für Video besorgt und erstmal einen virtuellen Taster hinterlegt zum Testen.
Was muss man in der Konfiguration unter "Code" eintragen? im Log steht nicht viel aber das Gerät taucht nicht auf in HomeKit.

p.s. ich nutze folgendes FFmpeg auf der CCU3: https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-armhf-static.tar.xz
Es lässt sich mal ausführen auf der Commandline, mehr testen konnte ich nicht

**Comment by Tom74HH** [comment]
moin, 

bin auch am Testen mit der Video Door Bell. Urls usw. hab ich alle zusammen. Bekomme das Gerät aber nicht in meinem HomeKit angezeigt. Anleitung wäre super.
